### PR TITLE
doc: document restriction about transparency for custom 3D objects

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: documentation/docs/examples/clipping.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  // Ping-pong plane between top of model and half-way to the bottom
+  const dt = (performance.now() - start) / 1000.0
+  const s = 0.5 * (Math.sin(Math.PI * dt / 10.0) + 1.0);
+  point.y = bbox.max.y - (bbox.max.y - bbox.min.y) / 2.0 * s;
+  plane.setFromNormalAndCoplanarPoint(orientation, point);
+
+  viewer.forceRerender();
+}, 1000 / 60);
+```
+:::tip
```
Comment (maksnester) [inline documentation/docs/examples/clipping.mdx:73]:
```suggestion
:::info
```

bluish info doesn't look so bright as that green tip.

File: documentation/docs/examples/clipping.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+mesh.position.copy(center);
+viewer.addObject3D(mesh);
+
+// Prepare a clipping plane
+const orientation = new THREE.Vector3(0, -1, 0);
+const point = new THREE.Vector3(0, center.y, 0);
+const plane = new THREE.Plane();
+plane.setFromNormalAndCoplanarPoint(orientation, point);
+viewer.setSlicingPlanes([plane]);
+
```
Comment (maksnester) [inline documentation/docs/examples/clipping.mdx:101]:
```suggestion
```

I know I didn't set up the linter everywhere, but still, if you have prettier installed, I believe you can configure it in your editor or at least run when you press some hotkey. So jsx will be formatted too.

File: documentation/docs/examples/clipping.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+:::
+
+### CAD geometry filter
+
+```js runnable
+// import { DragControls } from 'three/examples/jsm/controls/DragControls';
+
+const bounds = model.getModelBoundingBox();
+
+
```
Comment (maksnester) [inline documentation/docs/examples/clipping.mdx:87]:
```suggestion
```

File: documentation/docs/examples/clipping.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+transformControls.addEventListener('mouseDown', () => {
+  viewer.cameraControlsEnabled = false;
+});
+transformControls.addEventListener('mouseUp', () => {
+  viewer.cameraControlsEnabled = true;
+  viewer.setSlicingPlanes([plane]); // Trigger load after manipulation ends
+});
+transformControls.attach(mesh);
+viewer.addObject3D(transformControls);
+*/
```
Comment (maksnester) [inline documentation/docs/examples/clipping.mdx:130]:
```suggestion
```

Why these are comments?

Comment (larsmoa) [reply documentation/docs/examples/clipping.mdx:129]: Hm, though this PR still was a draft...

File: documentation/docs/demos/Cognite3DViewerDemo.tsx [hunk 5]
```diff
@@ -43,6 +44,7 @@ export default function Cognite3DViewerDemo({
     addModel({ modelId, revisionId });
 
     window.viewer = viewer;
+    (window as any).DragControls = DragControls;
```
Comment (maksnester) [inline documentation/docs/demos/Cognite3DViewerDemo.tsx:47]: Why not to put it into custom scope?

**Review by maksnester — CHANGES_REQUESTED** [review]
Requesting changes just to notify about comments

File: documentation/docs/demos/Cognite3DViewerDemo.tsx (hunk 5 above)
Comment (larsmoa) [reply documentation/docs/demos/Cognite3DViewerDemo.tsx:47]: Not sure how this ended up here as it's part of the clipping example, but how do I provide a custom scope @maksnester ?

Comment (maksnester) [reply documentation/docs/demos/Cognite3DViewerDemo.tsx:47]: with the customScope file

**Review by maksnester — APPROVED** [review]

**Pull request merged**